.github/workflows/_nodejs.condo.core.tests.yml

```diff
@@ -17,24 +17,37 @@ jobs:
     runs-on: ${{ inputs.runs-on }}
 
     steps:
+      - name: install deps
+        run: |
+          sudo apt update && sudo apt install -y git
+
+      - name: setup docker compose
+        uses: KengoTODA/actions-setup-docker-compose@v1
```

Comment:
R u sure we can trust this one? Seems like it's not that popular

Reply:
I think we can. But your truth, that's not a really popular solution to resolve installation issue. So, I can rollback to manual plugin installation process, if this is necessary